NEWS.md

```diff
@@ -1,5 +1,7 @@
 # rsconnect 0.8.30 (development version)
 
+* New `standardizeAppFiles()`, which supsersedes `listBundleFiles()`.
```

Comment:
This isn't my favorite name for a public function, but I don't have great suggestions. Are `enumerateTargetFiles` or `listTargetFiles` too similar to `listBundleFiles`?

Should the documentation hint about the common pattern for Rmd, which uses `rmarkdown::find_external_resources` to enumerate its requirements and populate `appFIles`?

Reply:
Something like `listDeploymentFiles` feels most fitting. I would like to eventually use "app" less. Use "content" where we must, but in contexts like this one, it does not feel necessary.